modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -71,5 +71,17 @@
 			"java",
 			"jsp"
 		]
+	},
+	{
+		"classNames": [
+			"CommercePriceEntryLocalServiceUtil",
+			"CommercePriceEntryLocalService"
```

Comment:
@alberttgomes please sort this, CommercePriceEntryLocalService should come first.